### ISSUE TITLE
Display path-to-notebook 

### ISSUE THREAD

**Issue by Zsailer** [jupyter/jupyterlab#1079]
JupyterLab is fantastic! It's a major (and impressive) upgrade to the NotebookApp UI. I've been super happy with the Alpha and look forward to what lies ahead. 

I am finding it a bit difficult, however, to keep track of where **running notebooks** live on my filesystem. In the old Notebook App, the paths-to-notebooks were visible in the URL and in the "Running" tab. In both cases, they are no longer visible (for good reasons, of course). This can be annoying if your "Jupyter Power Score" is high (mentioned here #573) and a bunch of "Untitled##.ipynb" are spread across your filesystem.

It would be nice to have the path-to-notebook displayed somewhere within JupyterLab. Perhaps, have it pop-up when you hover your mouse over the notebook name in the running notebook list or the notebook tab? Or have it displayed under the notebook tool bar?

**Comment by eskirk** [comment]
Hovering over a file tab in the lab actually displays the path to that file. Kinda a lil hidden feature, just to keep a lil sense of discovery in our hearts 
<img width="614" alt="screen shot 2016-10-12 at 10 12 32 am" src="https://cloud.githubusercontent.com/assets/7052378/19320061/9802188e-9064-11e6-99c2-7798a1a0798f.png">

**Comment by blink1073** [comment]
@Zsailer, thanks for the feedback!  The full path is also shown in the running tab as well (in master).

**Comment by Zsailer** [comment]
hey, thanks y'all! That's great!

@eskirk For some reason, the hovering feature wasn't working temporarily in my browser. I restarted chrome and it works! Thanks!

@blink1073 Ah, ok! Looks like I spoke too soon. I'm running lastest pip -- not master. I looked for mentions in the issues/PRs and (silly me) didn't check master. Sorry about that and thanks again!

I should mention, we're using this with Jupyterhub in my lab, and its been working great! Thanks for all the hard work here! 

Closing!
